019-restruture-the-installation-files.md

```diff
@@ -0,0 +1,64 @@
+# Restructure the installation files
+
+This proposal suggests restructuring to the folders of the [`strimzi-kafka-operator` GitHub repository](https://github.com/strimzi/strimzi-kafka-operator).
+
+## Current situation
+
+Currently, the Strimzi installation files in our master branch correspond to the images built from the master branch.
+That means they are under constant development and often work only with the freshly pulled images.
+That suites well for development work, but not necessarily for the users.
+They often checkout the GitHub repository, see the install or example folders and try them, not realising that they're using `master` rather than a stable release.
+If they just use them for a short time it should not matter.
+But if they keep using them, they will run into issues later.
+
+## Motivation
+
+Improve the experience of the users who checkout Strimzi and want to install / try it.
+
+## Proposal
+
+We should change the way we manage these files.
+This proposal suggests following:
+* Create a new subdirectory `packaging`
+    * This subdirectory will contain the _in-development_ versions of `install`, `examples` and `helm-chart` directories.
+    * Any changes in regular PRs, generated CRD files etc. will be done in this directory
+    * New releases will use these files for packaging
+* The original `install`, `examples` and `helm-chart` directories will be kept, but will contain the files for the latest stable release
+    * Updates to them will be done only during / after release
+    * They will not be changed with regular PRs
+
+### Risks
+
+The proposal keeps existing directories with more or less the same files but changes their purpose.
+That can be confusing for people using them today for the right purpose.
+It might be also confusing for new contributors who might try to change them in their PRs
```

Comment:
I think these very good points, I think we can mitigate this somewhat by adding/injecting comments into these files to make clear their purpose, like the install files could have large bannered comments saying 
"Release files - DO NOT EDIT" or something equally scary

Reply:
I was more thinking about having some `README.md` there with explanation. The `DO NOT EDIT` warning is tricky since users are actually expected to edit some of these to change namespaces or the default configuration.